00-HTML-CSS-basics/imageVideoSound.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="Topic 0 - Globant UI/CSS lab">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Topic 0</title>
+    
+</head>
+<body>
+    <header>
+        
+    </header>
+        <img width="560" height="315" src="http://dxtv.gob.ar/wp-content/uploads/jordan-1.jpg" alt="Jordan23">
+        <iframe width="560" height="315"
```

Comment:
do not use `iframe` since it is old, use `video` instead. ALso add different sources to have a little of browser compatibility